ci-operator/step-registry/baremetalds/metallb-e2e/baremetalds-metallb-e2e-workflow.yaml

```
@@ -7,5 +7,6 @@ workflow:
       - ref: baremetalds-metallb-e2e-test
     post:
       - chain: baremetalds-ipi-post
+    TEAM_NAME: ipi-metal
```

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/21071/pull-ci-openshift-release-master-pj-rehearse/1427632889662541824

```suggestion
    env:
      TEAM_NAME: ipi-metal
```